src/common/commandExecutor.ts

```diff
@@ -211,4 +214,19 @@ export class CommandExecutor {
     private generateRejectionForCommand(command: string, reason: any): Q.Promise<void> {
         return Q.reject<void>(ErrorHelper.getNestedError(reason, InternalErrorCode.CommandFailed, command));
     }
+
+    private static getCommandStatusString(command: string, status: CommandStatus) {
+        console.assert(status >= CommandStatus.Start && status <= CommandStatus.End, "Unsupported Command Status");
+
+        switch (status) {
+            case CommandStatus.Start:
+                return `Executing command: ${command}`;
+
+            case CommandStatus.End:
+                return `Finished executing: ${command}`;
+
+            default:
+                throw new Error("Unsupported command status");
```

Comment:
Because we have [switch-default](https://github.com/Microsoft/vscode-react-native/blob/master/tslint.json#L75) rule in our tslint.json
I could remove console.assert if you want it.

Reply:
Yes, please 👍

Reply:
Done